static/include/components/arm-services.md

```diff
@@ -0,0 +1,35 @@
+<div class="cards max-page">
+  <div class="col">
+    <div class="col sectionlist">
+       <ul class="sectionlist">
+       <li>
+        <a href="/services/frame-system/" target="_blank" title="The Robot Frame System">
+            <div>
+                <picture>
+                    <img src="/services/icons/frame-system.svg"
+                        width="/services/icons/frame-system"
+                        height="/services/icons/frame-system"
+                        alt="Frame System"
+                        loading="lazy">
```

Comment:
You need relative links to make this work on staging:
```suggestion
                    <img src="../../services/icons/frame-system.svg"
                        alt="Frame System"
                        loading="lazy">
```

while this does work I think you'd want this html to be generated automatically with you just typing something like:
```
{{< cards >}}
{{% itemcard link="/services/frame-system/" %}}
{{% itemcard link="/services/motion/" %}}
{{</ cards >}}
```

Let's have a look at it together when you get in - ping me when you see this :)